pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_agent.py

```diff
@@ -154,15 +179,23 @@ async def streamed_response():
         )
         activities.append(self.event_stream_handler_activity)
 
+        # Use wrapped_model if available, otherwise first registered model
+        if wrapped_model is not None:
+            primary_model = wrapped_model
+        else:
+            primary_model = next(iter(self._models_by_id.values()))
```

Comment:
This can be `wrapped_model or ...` right? Although that'll change if we do what I wrote above and don't require a model up front at all